documentation/src/main/asciidoc/topics/ref_hotrod_remote_cache.adoc

```diff
@@ -48,7 +49,7 @@ have to copy the contents yourself. Also you can do this with `entrySet` and
 Set<K> keysCopy = remoteCache.keySet().stream().collect(Collectors.toSet());
 ----
 
-Please use extreme cautiong with this as a large number of keys can and will cause
+Please use extreme caution with this as a large number of keys can and will cause
 OutOfMemoryError in the client.
 
 [source,java]
```

Comment:
this example can be removed IMO. it didn't add much to the docs.

Reply:
thanks. I've removed it.